diesel/src/migrations/migration_error.rs

```diff
@@ -12,6 +15,23 @@ pub enum MigrationError {
     UnknownMigrationVersion(String),
 }
 
+impl Error for MigrationError {
+    fn description(&self) -> &str {
+        match *self {
+            MigrationDirectoryNotFound => "Unable to find migrations directory in this directory or any parent directories.",
+            UnknownMigrationFormat(_) => "Invalid migration directory, the directory's name should be <timestamp>_<name_of_migration>, and it should only contain up.sql and down.sql.",
+            IoError(ref error) => error.description(),
+            UnknownMigrationVersion(_) => "Invalid migration version, only YYYYMMDDHHMMSS format is supported.",
```

Comment:
That's not what this means. It occurs during revert if we can't find the migration on disk that the database says we should revert